src/app/(auth)/login/page.tsx

```diff
@@ -0,0 +1,38 @@
+import { login } from "@/auth/login";
+import Link from "next/link";
+
+export default function LoginPage() {
+  return (
+    <div className="absolute left-[50%] top-[50%] w-[25vw] -translate-x-2/4 -translate-y-1/2 justify-center rounded-md bg-white/25 p-10">
```

Comment:
I wouldn't use position absolute here, you're better creating a wrapper that has a height h-screen and giving the login page m-auto